lib/ui/LandingPage.dart

```diff
@@ -0,0 +1,236 @@
+            fit: FlexFit.loose,
+            child: Container(
+              child: Image.asset(
+                _viewModel.headerImage,
+                width: double.infinity,
+              ),
+            ),
+          ),
+          Container(
+            height: _Constants.bottomContainerHeight,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: <Widget>[
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: <Widget>[
+                    Image.asset(_viewModel.subtitleImage),
+                    SizedBox(
+                      height: _Constants.subtitleLineSpace,
+                    ),
+                    Text(
+                      Intl.message(_viewModel.detailText),
+                      style: _style.detailTextStyle,
+                      maxLines: _style.detailTextMaxLines,
+                      overflow: TextOverflow.ellipsis,
+                      textAlign: TextAlign.center,
+                    ),
+                  ],
+                ),
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: <Widget>[
+                    RoundedButton(
+                      viewModel: RoundedButtonViewModel(
+                        title: Intl.message(_viewModel.actionText),
+                        onTap: () => _viewModel.continueAction(),
+                      ),
+                      style: RoundedButtonStyle.largeRoundedButtonStyle()
+                          .copyWith(buttonTextStyle: _style.actionTextStyle),
+                    ),
+                    SizedBox(
+                      height: _Constants.actionLineSpace,
+                    ),
+                    GestureDetector(
+                      behavior: HitTestBehavior.opaque,
+                      onTap: () => _languageTapped(_languageMenu(), context),
+                      child: Container(
+                        margin: _Constants.actionTapArea,
+                        child: Text(
+                          Intl.message(_viewModel.footerText),
+                          style: _style.footerTextStyle,
+                          maxLines: _style.footerTextMaxLines,
+                          overflow: TextOverflow.ellipsis,
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+              ],
+            ),
+          )
+        ],
+      ),
+    );
+  }
+
+  void _languageTapped(ActionSheet sheet, BuildContext context) {
+    ActionSheet.present(sheet, context);
+  }
+
+  ActionSheet _languageMenu() {
+    final actions = [
+      ActionSheetListItemViewModel(
+        title: Intl.message(_Strings.englishAction),
+        type: ActionType.selectable,
+        icon: _Icons.englishIcon,
+        checkBoxIcon: _Icons.checkBoxIcon,
+        onTap: () => _viewModel.switchLanguageTapped(_Languages.english),
```

Comment:
nice,    @DavidFD005 has a PR which has the real function to switch. would be good to sync up to hook this up in another PR.